src/features/FavoriteItem/view/FavoriteItem.jsx

```diff
@@ -0,0 +1,5 @@
+function FavoriteItem() {
```

Comment:
The same like with Favorite. If this component contains cities, name it FavoriteCity